cmd/immudb/command/cmd_test.go

```diff
@@ -137,7 +139,7 @@ func TestImmudbCommandFlagParserPriority(t *testing.T) {
 	assert.Equal(t, "ConfigFileThatsNameIsDeclaredOnTheCommandLine", options.Logfile)
 
 	// 3. env. variables
-	os.Setenv("IMMUDB_LOGFILE", "EnvironmentVars")
+	t.Setenv("IMMUDB_LOGFILE", "EnvironmentVars")
```

Comment:
I learned another thing today. Of course the testing package would have a function for that. Unfortunately it cannot be used in parallel tests, but that's the same for `os.Setenv`

Reply:
TBH I only learned about this method while doing this cleanup PR ;) Wrote a utility method first to figure out that it's already there shortly after.